server/views/super/logs.twig

```diff
@@ -5,6 +5,6 @@
 {% endblock %}
 
 {% block script %}
-    <script src="/super/client/dist/logs.js"></script>
+    <script src="{{ asset_url }}"></script>
```

Comment:
How about `{{ asset('logs.js') }}`?
It's much better form since backend is not responsible for the context.